lib/configs.json

```diff
@@ -0,0 +1,15 @@
+{
+    "constants": {
```

Comment:
Fixed in PR https://github.com/wso2/carbon-uuf/pull/2 (see commit https://github.com/sajithar/carbon-uuf/commit/5f7fefaefb88b0c424c61074b1890dba7218d2ed)